index.html

```diff
@@ -64,5 +64,196 @@ <h2 class="intro connect">LET'S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <!-- Work Section -->
+    <section class="wrapper">
+      <div class="shot">
+        <div class="snapshot"><img src="images/Shot1.svg" alt="tonic" /></div>
+        <div class="block">
+          <h1 class="heading1">Tonic</h1>
```

Comment:
- [ ] You should use only one `<h1>` element for semantic reasons and that one should be used with the most significant heading you have in that particular HTML page. In this case, the most important heading is in the headline section. Therefore, kindly replace all other `<h1>` tags with `<h2>` to `<h6>` hierarchically.